Project EFT/Database/DBConnector.cs

```diff
@@ -153,6 +154,20 @@ public static int UpdatePassword(User user, string newPassword)
             return result == 1 ? CREDENTIAL_CHANGE_SUCCESS : DB_FAILURE;
         }
 
+        public static int UpdateProblem(Problem problem)
+        {
+            MySqlCommand command = MakeCommand("UPDATE Problems SET Problem_Title = @title, Problem_Answer = @answer, Problem_Attempts = @attempts, Problem_Completions = @completions WHERE Problem_Number = @id");
+            command.Parameters.AddWithValue("@title", problem.Title);
+            command.Parameters.AddWithValue("@answer", problem.Answer);
+            command.Parameters.AddWithValue("@attempts", problem.Attempts);
+            command.Parameters.AddWithValue("@completions", problem.Completions);
+            command.Parameters.AddWithValue("@id", problem.ProblemNumber);
+            command.Prepare();
+            int result = command.ExecuteNonQuery();
+
+            return result;
+        }
```

Comment:
presumably, this will only be used for updating attempts / completions, so is it necessary to also 'update' the title, answer, etc.?

Reply:
I was considering long term where we may want a version of the admin page that updates a problem, because someone mistypes the answer to the cipher, or a bad admin adds a crap problem, something like that.  Rather than just deleting it, we would just update that problem.  I can edit it if you feel its necessary, but I just felt it might be better to have the more general version.